corgi/core/models.py

```diff
@@ -876,6 +817,22 @@ def srpms(self) -> models.QuerySet["Component"]:
     def root_components(self) -> models.QuerySet["Component"]:
         return self.filter(ROOT_COMPONENTS_CONDITION)
 
+    def latest_components(self) -> models.QuerySet["Component"]:
+        """Return latest versions of components in some queryset."""
+        return self.annotate(
+            latest=models.Subquery(
+                self.filter(
+                    name=models.OuterRef("name"),
+                    el_match=models.OuterRef("el_match"),
+                )
+                .order_by(
```

Comment:
what I do not understand is why we are no longer using software build completion date in this calculation ...

Reply:
for safety ... I propose we keep latest filter we have today asis (at least until Jason gets back and gets a chance to review the rationale of this change) - and provide an OSC filter as you have defined (which makes sense for them) ... it does not matter which you decide to make the default (or how they are named) as long as we can reach either of them in a sensible manner. make sense ?